src/jsifier.js

```diff
@@ -244,7 +244,9 @@ function JSify(data, functionsOnly) {
       } else if (typeof snippet === 'function') {
         isFunction = true;
         snippet = processLibraryFunction(snippet, ident, finalName);
-        if (ident[0] != '$') { // Functions that start with '$' should not be imported to asm.js/wasm module
+        // Functions that start with '$' should not be imported to asm.js/wasm module, since they are
+        // intended to be exclusive to JS code only.
```

Comment:
@sbc100 also added that such things cannot come from C anyhow, as `$` is not a valid identifier there, so the convention is a natural one. lgtm with that.